src/sbt-test/windows/test-bat-template/build.sbt

```diff
@@ -114,18 +114,16 @@ TaskKey[Unit]("checkScript") := {
   checkOutput("include symbols on -D", Seq("-Dtest.hoge=\\[]!< >%"), "property(test.hoge) is [\\[]!< >%]\nSUCCESS!")
   checkOutput("include symbols on normal args", Seq("\"\\[]!< >%\""), "arg #0 is [\\[]!< >%]\nSUCCESS!")
 
-  /* fails test because symbols '<' and '>' cannot be properly escaped during cmd execution
   checkOutput(
     "include symbols with double quote",
     Seq("-Dtest.huga=\"[]!<>%\""),
     "property(test.huga) is [[]!<>%]\nSUCCESS!"
   )
-  */
 
   checkOutput(
     "include symbols with double quote2",
-    Seq("-Dtest.hoge=\\[]!< >%", "\"\\[]!< >%\"", "-Dtest.huga=\\[]!<>%"),
-    "arg #0 is [\\[]!< >%]\nproperty(test.hoge) is [\\[]!< >%]\nproperty(test.huga) is [\\[]!<>%]\nSUCCESS!"
+    Seq("-Dtest.hoge=\\[]!< >%", "\"\\[]!< >%\"", "-Dtest.huga=\\[]!%"),
+    "arg #0 is [\\[]!< >%]\nproperty(test.hoge) is [\\[]!< >%]\nproperty(test.huga) is [\\[]!%]\nSUCCESS!"
```

Comment:
So, I found what made the test fail on AppVeyor running Windows.
For the record, the error message was:
```
error < > was unexpected at this time.
```

Turns out, this has nothing to do with this pull request per se, you can easily reproduce this error even with sbt-native-packager 1.9.16 on the current main branch, without this PR applied:
1. You have to be on Windows
1. Checkout the current `main` branch 
1. Go to the failing scripted test's directory:
  ```
  cd src/sbt-test/windows/test-bat-template
  ```
1. The test succeeds with sbt 1.5.8:
  ```
  sbt --sbt-version 1.5.8 -Dproject.version=1.9.16
  sbt:windows-test> stage
  sbt:windows-test> checkScript
  # The test works
  ```
1. The test fails starting with sbt 1.6.0-M1:
  ```
  sbt --sbt-version 1.6.0-M1 -Dproject.version=1.9.16
  sbt:windows-test> stage
  sbt:windows-test> checkScript
  # The test fails with above error message
  ```

I did check and debug: The process args and environment passed [here](https://github.com/sbt/sbt-native-packager/blob/v1.9.19/src/sbt-test/windows/test-bat-template/build.sbt#L59) are 100% the same for both sbt 1.5.8 and 1.6.0-M1, nothing changed... So why does sbt 1.6+ fail?
Before we go deeper, the technical reason why it fails is because `<` and `>` are handled like normal pipes starting with sbt 1.6+ and not passed as string.
So why is that?
Let's look at the Windows specific code that runs a process, [this line](https://github.com/AdoptOpenJDK/openjdk-jdk8u/blob/master/jdk/src/windows/classes/java/lang/ProcessImpl.java#L389). You can see there is a check for a `SecurityManager`, and based on its existence things [behave differently](https://github.com/AdoptOpenJDK/openjdk-jdk8u/blob/master/jdk/src/windows/classes/java/lang/ProcessImpl.java#L393).
So if a `SecurityManager` is available, the process will be started differently: the process args will be handled and escaped differently.
Now.. it turns out until sbt 1.5.8, sbt did install a `SecurityManager`, but starting with sbt 1.6.0-M1 it does not anymore.
Look at [this commit](https://github.com/sbt/sbt/commit/0413727796adebdb25d9fdf70ea038723f354ee6) in sbt, you will see in `Main.scala` the `TrapExit.installManager()` got removed.
So this explains why the test started to fail with sbt 1.6.+

Now the thing is, assuming most people upgraded to sbt 1.6 or newer already anyway, this test now just got adjusted to the current real world sbt behaviour.
The reason why the scripted test fails with this pull request is because I upgraded sbt to 1.9.9 and by default the scripted tests are run with the sbt version of the main project. Like showed above however, this behaviour already surfaces in any project that uses sbt-native-packager with sbt 1.6+.
(I mean in theory I could make the scripted test here keep running with sbt 1.5.4, so I wouldn't have to adjust the test itself, but that does not make any sense IMHO).

Reply:
Wow, TIL. I agree it doesn't make sense to depend on deprecated `SecurityManager` behavior.

Reply:
> Wow, TIL.

Yeah, TIL too, took me hours to figure out what the hell is going on...